data/hts/edgt_lyon/filtered.py

```diff
@@ -6,7 +6,15 @@
 """
 
 def configure(context):
-    context.stage("data.hts.edgt_lyon.cleaned")
+    edgt_lyon_source = context.config("edgt_lyon_source", "adisp")
+
+    if edgt_lyon_source == "adisp":
+        context.stage("data.hts.edgt_lyon.cleaned_adisp", alias="data.hts.edgt_lyon.cleaned")
+    elif edgt_lyon_source == "cerema":
+        context.stage("data.hts.edgt_lyon.cleaned_cerema", alias="data.hts.edgt_lyon.cleaned")
+    else:
```

Comment:
And then here another elif that raises an exception, but explicitly telling the user that the EDGT source needs to be chosen, if the value is still "unchosen". Then we can nicely remind the user to set it :)